### DISCUSSION TITLE
P2P

### DISCUSSION THREAD

**Discussion by meduar** [pion/webrtc#2260] [General] [answered]
I spent a lot of time setting up Janus until I realized it doesn't work for P2P connections since the signaling is done inside Janus, so my questions are:

- Pion supports p2p?
- how is the signaling done(I am ok doing my own signaling server)

Basically, I need to transfer the video from a camera(RTSP) (iot device -MIPS) -> webrtc ->to a REMOTE (Web app) or NAt to Nat, NOT port forwarding

Would PION help me to build it? any docs or examples to do a P2P connection?

Thank you,

**Comment by meduar** [comment, marked as answer]
Found the answer

**Comment by Rahul-ambiplatforms** [comment]
Please Share answer

i have mips architecture camera i want to build p2p connectivity for rtsp streaming
